tests/test_braid_vec.py

```diff
@@ -0,0 +1,684 @@
+from functools import partial
+import pytest
+import numpy as np
+
+# IMPORTANT: knpy should be installed first
+from knpy.braid_vec import Braid
+# from knpy.braid import BraidTransformation
+from knpy import IllegalTransformationException, InvalidBraidException, IndexOutOfRangeException
+
+
+class TestBraidClassBraidRelationsInit:
+    def test_init_empty(self) -> None:
+        braid = Braid([])
+        assert braid.strand_count == 1
+        assert braid.notation().shape[0] == 0
```

Comment:
Please apply changes from merges also applied to `test_braid.py`. Here: update tests to include types.

Reply:
Done